bundles/org.openhab.binding.fritzboxtr064/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="fritzboxtr064"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="rootdevice">
+		<label>Fritzbox Root-Device</label>
```

Comment:
I'd think this label would be what most people will understand:
```suggestion
		<label>DSL Router</label>
```

Reply:
FB 6360/6490/6591 are Cable routers, so DSL would be the wrong label

Reply:
Ok, good point. I was reading https://avm.de/service/schnittstellen/, where AVM states:
"TR-064 ist ein vom DSL-Forum entwickeltes Protokoll, um DSL-Internetzugangsgeräte aus dem lokalen Netz zu konfigurieren."
I was somehow thus expecting that it is only used in the context of DSL routers. I guess AVM just kept it when they accidentially also produced cable routers ;-).
So let's go for "Router" or "Internet Gateway" or something like that then.